web/gui/dashboard_info.js

```diff
@@ -2789,6 +2789,104 @@ netdataDashboard.context = {
         '<b>Metadata</b> is bcache\'s metadata overhead. '
     },
 
+    // ------------------------------------------------------------------------
+    // NFS client
+
+    'nfs.net': {
+        info: 'The number of received UDP and TCP packets.'
+    },
+
+    'nfs.rpc': {
+        info: '<p>Remote Procedure Call (RPC) statistics.</p>'+
+        '</p><b>Calls</b> - all RPC calls. '+
+        '<b>Retransmits</b> - retransmitted calls. '+
+        '<b>AuthRefresh</b> - authentication refresh calls (validating credentials with the server).</p>'
+    },
+
+    'nfs.proc2': {
+        info: 'NFSv2 RPC calls. The individual metrics are described in '+
+        '<a href="https://datatracker.ietf.org/doc/html/rfc1094#section-2.2" target="_blank">RFC1094</a>.'
+    },
+
+    'nfs.proc3': {
+        info: 'NFSv3 RPC calls. The individual metrics are described in '+
+        '<a href="https://datatracker.ietf.org/doc/html/rfc1813#section-3" target="_blank">RFC1813</a>.'
+    },
+
+    'nfs.proc4': {
+        info: 'NFSv4 RPC calls. The individual metrics are described in '+
+        '<a href="https://datatracker.ietf.org/doc/html/rfc1813#section-3" target="_blank">RFC1813</a>.'
```

Comment:
To provide more information about NFSv4 operations [rfc7530](https://datatracker.ietf.org/doc/html/rfc7530#section-16) (NFSv4.0) and [rfc7862](https://datatracker.ietf.org/doc/html/rfc7862/#section-15) (NFSv4.2) should be added as well, but [rfc8881](https://datatracker.ietf.org/doc/html/rfc8881#section-18) (NFSv4.1) covers most of the operations.
```suggestion
        '<a href="https://datatracker.ietf.org/doc/html/rfc8881#section-18" target="_blank">RFC8881</a>.'
```